packages/matchbox/src/components/Panel/Header.js

```diff
@@ -1,22 +1,46 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { buttonsFrom } from '../Button';
-import styles from './Panel.module.scss';
+import styled from 'styled-components';
+import { actions, header, headerText } from './styles';
+import { padding } from 'styled-system';
+import { createPropTypes } from '@styled-system/prop-types';
+
+const HeaderOuter = styled('div')`
+  ${header}
+  ${padding}
+`;
+
+export const HeaderText = styled('div')`
+  ${headerText}
+`;
+
+const Actions = styled('div')`
+  ${actions}
+`;
 
 const actionOverrides = { flat: true, size: 'small' };
 
-const Header = ({ title, actions }) => {
-  const actionMarkup = actions && actions.length
-    ? <div className={styles.Actions}>{buttonsFrom(actions, actionOverrides)}</div>
-    : null;
+const Header = ({ title, actions, ...rest }) => {
+  const actionMarkup =
+    actions && actions.length ? <Actions>{buttonsFrom(actions, actionOverrides)}</Actions> : null;
```

Comment:
This is actually a really good place to use the new `Inline` component, same with the actions inside sections

Reply:
Actually, nevermind, this produces a button group, not buttons